src/types.ts

```diff
@@ -0,0 +1,13 @@
+enum createEventReducerTypes {
+  eventTitle = 'EVENT_TITLE',
+  evaluators = 'EVALUATORS',
+  evaluatees = 'EVALUATEES',
+  criterias = 'CRITERIAS',
```

Comment:
Wait to @yavetisyan PR to be fixed or work with him to fix it then migrate this part to there